src/pages/ProductPage.module.css

```diff
@@ -0,0 +1,39 @@
+.MainProductPage {
+  max-width: 100%;
+  max-height: 100%;
+  margin-bottom: 100px;
+}
+.ProductPageContainer {
+  text-align: center;
+  max-width: 1120px;
+  margin: 0 auto;
+}
+.titlePage {
+  display: flex;
+  justify-content: space-between;
+}
+.IconButton {
+  margin-top: 40px;
+}
+.ProductDescriptionContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+.ProductDescription {
+  flex-basis: 50%;
+}
```

Comment:
Left column has near 560px. Maybe if you change flex-basis in .ProductDescription to 58% it will be near 650px that is more similar as in AirBnB(653px) and it can reduce the gap between ProductDescription and Reservation card.

Reply:
I believe it's not needed to be 100% similar to the AirBnB website, as several components that have been created are not 100% similar...